src/guides/v2.3/comp-mgr/patching.md

```diff
@@ -24,11 +18,17 @@ Patch (or diff) files are text files that note:
 
 When the [patch][] program is run, this file is read in and the specified changes are made to the file(s).
 
+There are three types of patches:
```

Comment:
If you say three, should it be a numbered list?

Reply:
I think that's subjective. I prefer using ordered lists when sequence is important.